core/src/player.rs

```diff
@@ -2342,9 +2342,9 @@ impl PlayerBuilder {
         let mut player_lock = player.lock().unwrap();
         player_lock.mutate_with_update_context(|context| {
             // Instantiate an empty root before the main movie loads.
-            let fake_root = MovieClip::new(fake_movie, context.gc_context);
-            fake_root.post_instantiation(context, None, Instantiator::Movie, false);
-            context.stage.replace_at_depth(context, fake_root.into(), 0);
+            //let fake_root = MovieClip::new(fake_movie, context.gc_context);
```

Comment:
Are these changes intentional, or left over from a previous PR?

Reply:
This is https://github.com/ruffle-rs/ruffle/pull/11083

Reply:
My changes previously needed this change, but with the latest version of this PR, all of the tests pass with the fake root in place. I've re-added this code.